### PR TITLE
mgr/rook: Set default nfs daemon count to 1

### PR DESCRIPTION
**Pull request by varshar16**
Fixes: https://tracker.ceph.com/issues/47924
Signed-off-by: Varsha Rao <varao@redhat.com>

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/rook/module.py [hunk 1]
```diff
@@ -415,6 +415,8 @@ def _service_add_decorate(self, typename, spec, func):
 
     def add_nfs(self, spec):
         # type: (NFSServiceSpec) -> RookCompletion
+        if spec.placement.count == None:
```
Comment (travisn) [inline src/pybind/mgr/rook/module.py:418]: The error in the tracker is 
```
Invalid value: "null": spec.server.active in body must be of type integer
```
Should this instead be setting `spec.server.active`?

Comment (varshar16) [reply src/pybind/mgr/rook/module.py:418]: Here `spec.server.active` is initialized with `spec.placement.count` value.

https://github.com/ceph/ceph/blob/master/src/pybind/mgr/rook/rook_cluster.py#L469-L474

Comment (travisn) [reply src/pybind/mgr/rook/module.py:418]: I see, this is a different object.

**Review by travisn — APPROVED** [review]

**Comment by varshar16** [comment]
https://shaman.ceph.com/builds/ceph/wip-rook-nfs-fix-daemon-default-cnt/

**Comment by varshar16** [comment]
PR https://github.com/ceph/ceph/pull/37881 fixes this issue.

**Pull request closed without merging**